doc/tools/coqrst/coqdomain.py

```diff
@@ -1108,4 +1144,7 @@ def setup(app):
     # Tell Sphinx about extra settings
     app.add_config_value("report_undocumented_coq_objects", None, 'env')
 
-    return {'version': '0.1', "parallel_read_safe": True}
+    meta = { "version": "0.1",
+             "env_version": 1, # Increment this when the contents of env are changed
```

Comment:
I'm not sure where this `env` is defined or what will happen if someone forgets to increment this.

Reply:
I improved the comment